### ISSUE TITLE
nosetests fail with several errors

### ISSUE THREAD

**Issue by jmelcr** [theochem/horton#151]
Hi,

after installing Horton on my Ubuntu 16.04, I get following nostests errors:

```python
EEEEE...............................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................EE...............................................E.....................................................................................................................................................
======================================================================
ERROR: horton.correlatedwfn.test.test_1dm_ap1rog.test_ap1rog_one_dm
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/home/joe/.local/lib/python2.7/site-packages/horton/correlatedwfn/test/test_1dm_ap1rog.py", line 60, in test_ap1rog_one_dm
    geminal_solver = RAp1rog(lf, occ_model)
NameError: global name 'RAp1rog' is not defined

======================================================================
ERROR: horton.correlatedwfn.test.test_geminals.test_ap1rog_cs
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/home/joe/.local/lib/python2.7/site-packages/horton/correlatedwfn/test/test_geminals.py", line 69, in test_ap1rog_cs
    geminal_solver = RAp1rog(lf, occ_model)
NameError: global name 'RAp1rog' is not defined

======================================================================
ERROR: horton.correlatedwfn.test.test_geminals.test_ap1rog_cs_scf
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/home/joe/.local/lib/python2.7/site-packages/horton/correlatedwfn/test/test_geminals.py", line 80, in test_ap1rog_cs_scf
    geminal_solver = RAp1rog(lf, occ_model)
NameError: global name 'RAp1rog' is not defined

======================================================================
ERROR: horton.correlatedwfn.test.test_geminals.test_ap1rog_cs_scf_restart
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/home/joe/.local/lib/python2.7/site-packages/horton/correlatedwfn/test/test_geminals.py", line 91, in test_ap1rog_cs_scf_restart
    geminal_solver = RAp1rog(lf, occ_model)
NameError: global name 'RAp1rog' is not defined

======================================================================
ERROR: horton.correlatedwfn.test.test_lagrange.test_ap1rog_lagrange
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/home/joe/.local/lib/python2.7/site-packages/horton/correlatedwfn/test/test_lagrange.py", line 60, in test_ap1rog_lagrange
    geminal_solver = RAp1rog(lf, occ_model)
NameError: global name 'RAp1rog' is not defined

======================================================================
ERROR: Failure: ImportError (cannot import name get_fake_co)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/nose/loader.py", line 418, in loadTestsFromName
    addr.filename, addr.module)
  File "/usr/lib/python2.7/dist-packages/nose/importer.py", line 47, in importFromPath
    return self.importFromDir(dir_path, fqname)
  File "/usr/lib/python2.7/dist-packages/nose/importer.py", line 94, in importFromDir
    mod = load_module(part_fqname, fh, filename, desc)
  File "/home/joe/.local/lib/python2.7/site-packages/horton/part/test/test_cpart.py", line 26, in <module>
    from horton.part.test.common import check_names, check_proatom_splines, \
ImportError: cannot import name get_fake_co

======================================================================
ERROR: horton.part.test.test_hirshfeld_e.test_hebasis
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/home/joe/.local/lib/python2.7/site-packages/horton/part/test/test_hirshfeld_e.py", line 32, in test_hebasis
    hebasis = HEBasis(numbers, padb)
NameError: global name 'HEBasis' is not defined

======================================================================
ERROR: Failure: ImportError (cannot import name CPart)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/nose/loader.py", line 418, in loadTestsFromName
    addr.filename, addr.module)
  File "/usr/lib/python2.7/dist-packages/nose/importer.py", line 47, in importFromPath
    return self.importFromDir(dir_path, fqname)
  File "/usr/lib/python2.7/dist-packages/nose/importer.py", line 94, in importFromDir
    mod = load_module(part_fqname, fh, filename, desc)
  File "/home/joe/.local/lib/python2.7/site-packages/horton/scripts/test/test_cpart.py", line 29, in <module>
    from horton.scripts.cpart import cpart_schemes
  File "/home/joe/.local/lib/python2.7/site-packages/horton/scripts/cpart.py", line 38, in <module>
    cpart_schemes = get_cpart_schemes()
  File "/home/joe/.local/lib/python2.7/site-packages/horton/scripts/cpart.py", line 30, in get_cpart_schemes
    from horton.part.base import CPart
ImportError: cannot import name CPart

----------------------------------------------------------------------
Ran 1051 tests in 206.955s

FAILED (errors=8)
```

I did not dig very deep into it, but it looks like there are probably some small things forgotten during the common workflow.

BR,
Joe.

**Comment by tovrstra** [comment]
It seems that you are running examples of 2.0.0 or 2.0.1 with the master branch of HORTON. We have kicked out some (unmaintainable) code. These features are being developed in new packages outside the main HORTON repository but they are not released yet.

**Comment by jmelcr** [comment]
In other words -- nothing serious, tests of features, that weren't implemented yet. (?)

**Comment by tovrstra** [comment]
There were once implemented, so you can download the 2.0.x version and run the examples.

We are just in the process of reorganizing the code and some (problematic) parts was removed in that process.

**Comment by tovrstra** [comment]
Discussion is continued on #150. Sort-of-duplicate.
